libkbfs/folder_block_ops.go

```diff
@@ -160,6 +160,16 @@ func (fbo *folderBlockOps) GetState(lState *lockState) overallBlockState {
 	return dirtyState
 }
 
+func (fbo *folderBlockOps) getBlockFromCache(ptr BlockPointer,
```

Comment:
assuming you agree with the two comments below, then this function only has one caller, so may as well inline it (to avoid future cases where it would be better to check only one cache explicitly)